### DISCUSSION TITLE
LedgerConnector doesn't work on Tauri

### DISCUSSION THREAD

**Discussion by sumi0820** [wevm/wagmi#2256] [Q&A]
We're developing a desktop wallet app with [Tauri](https://tauri.app/). We tested wagmi's **LedgerConnector** to connect ledger hardware wallets, and a popup was displayed but there's an error message on the console saying **"_[Error] Failed to load resource: the server responded with a status of 404 () (index.js.map, line 0) : https://cdn.jsdelivr.net/npm/@ledgerhq/index.js.map_".**

Does this mean LedgerConnector is not working on Tauri as the same as Ledger's native libraries because the window is NOT chromium browsers? If so, is there any workaround to make it work in Tauri environment?
![Screenshot 2023-05-03 at 12 41 07](https://user-images.githubusercontent.com/48553998/235829027-2ec6d8a3-f24b-4bda-badb-8bf7535c0c6d.png)

**Comment by tmm** [comment]
Looks like the jsdelivr link shared above does not resolve to anything, thus the 404.
